storage/postgres/notificator.go

```diff
@@ -95,10 +100,11 @@ func (n *Notificator) Start(ctx context.Context, group *sync.WaitGroup) error {
 	n.setConnection(n.connectionCreator.NewConnection(func(isConnected bool, err error) {
 		if isConnected {
 			atomic.StoreInt32(&n.isConnected, aTrue)
+			log.C(n.ctx).Debug("db connection for notifications established")
 		} else {
 			atomic.StoreInt32(&n.isConnected, aFalse)
-			log.C(n.ctx).WithError(err).Info("connection to db closed, closing all consumers")
-			n.closeAllConsumers()
+			log.C(n.ctx).WithError(err).Error("DB connection for notifications closed")
+			n.stopProcessing()
```

Comment:
add godoc/log what this does because it's somewhat hidden (applies to every place calling `stopProcessing`)